vault/request_handling.go

```diff
@@ -1203,7 +1203,7 @@ func (c *Core) handleLoginRequest(ctx context.Context, req *logical.Request) (re
 			}
 
 			auth.EntityID = entity.ID
-			validAliases, err := c.identityStore.refreshExternalGroupMembershipsByEntityID(ctx, auth.EntityID, auth.GroupAliases)
+			validAliases, err := c.identityStore.refreshExternalGroupMembershipsByEntityID(ctx, auth.Alias.MountAccessor, auth.EntityID, auth.GroupAliases)
```

Comment:
auth.Alias.MountAccessor is set above on line 1180 (`auth.Alias.MountAccessor = req.MountAccessor`) and can be safely referenced here